### PR TITLE
Use arrow up/arrow down key to adjust attention weight in prompt input box

### LINKED ISSUE #170: [Feature Request] Use Up/Down arrow key to increase/decrease prompt keyword weight
It is a very handy A1111 feature that is missing in this extension right now. I can take a look and try to port the feature from A1111.

### PR DESCRIPTION
**Pull request by huchenlei**
Closes #170.

ezgif-4-8a57da64aa

Note:
There is a behaviour difference in multi vs single line widget, as there is a bug on `setSelection` in single line, which sometimes set the selection end to end of text when correct start/end pair are passed. Currently behaviour is multi widget creates a selection, while single widget moves the cursor.

### CODE REVIEW

File: ai_diffusion/attention_edit.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    while end < len(text) and text[end] not in delimiters:
+        end += 1
+
+    return start, end
+
+
+def select_on_cursor_pos(text: str, cursor_pos: int) -> Tuple[int, int]:
+    """Return a range in the text based on the cursor_position."""
+    return (
+        select_current_parenthesis_block(text, cursor_pos, "(", ")")
```
Comment (Acly) [inline ai_diffusion/attention_edit.py:54]: as far as I know, only `( )` parenthesis work in comfy

File: ai_diffusion/styles/default.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (Acly) [inline ai_diffusion/styles/default.json:1]: accidental commit

**Comment by huchenlei** [comment]
How about making the trigger ctrl+arrow keys? I think A1111 does that in a multi-line textbox.

**Comment by Acly** [comment]
> How about making the trigger ctrl+arrow keys? I think A1111 does that in a multi-line textbox.

That's also good. I just checked and it's Ctrl in Comfy's UI too, so likely people are used to it.

**Comment by huchenlei** [comment]
Changed to Ctrl + arrow key binding. I think the multi-line textbox now has no unexpected behaviours.

File: ai_diffusion/styles/default.json (hunk 2 above)
Comment (huchenlei) [reply ai_diffusion/styles/default.json]: File removed. Should we ignore the styles directory?

**Comment by Acly** [comment]
👍 Looks good

**Pull request merged**